optaplanner-core/src/main/java/org/optaplanner/core/impl/domain/locator/Locator.java

```diff
@@ -21,22 +21,20 @@
 import java.util.Map;
 
 import org.optaplanner.core.api.domain.locator.PlanningId;
-import org.optaplanner.core.api.domain.solution.PlanningSolution;
 import org.optaplanner.core.impl.domain.solution.descriptor.SolutionDescriptor;
 import org.optaplanner.core.impl.score.director.ScoreDirector;
 
 /**
- * @param <Solution_> the solution type, the class with the {@link PlanningSolution} annotation
  * @see PlanningId
  * @see ScoreDirector#locateWorkingObject(Object)
  */
-public class Locator<Solution_> {
+public class Locator {
 
     private final LocationStrategyResolver locationStrategyResolver;
 
     private Map<Object, Object> idToWorkingObjectMap;
 
-    public Locator(SolutionDescriptor<Solution_> solutionDescriptor) {
+    public Locator(SolutionDescriptor<?> solutionDescriptor) {
```

Comment:
I apologize, I didn't read your proposal of the constructor change carefully enough and thought you wanted to change the parameter to `SolutionDescriptor` (rawtype). So my previous comment doesn't make sense because it proposes exactly what you intended. Please ignore it :-)